Common/Models/WatchContext.swift

```diff
@@ -48,6 +48,8 @@ final class WatchContext: RawRepresentable {
 
     var cgmManagerState: CGMManager.RawStateValue?
 
+    var dosingEnabled: Bool?
```

Comment:
Added this to `WatchContext` because (for some reason) I'm not seeing it updating in `LoopSettings` on the Watch (in the simulator).